simple_switch/l2_switch_grpc/controller/main.cpp

```diff
@@ -10,7 +10,7 @@
 int main(int argc, char* argv[])
 {
     // TODO: Better command line parsing
-    if (argc != 6)
+    if (argc < 10 || argc > 11)
```

Comment:
This line should not have been changed in l2_switch_grpc

Reply:
Done.